.husky/post-checkout

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# Don't do it in CI
+# Branch checkout, not file checkout
+# Not a clone operation
+# Old ref and new ref are different (i.e. actually switched branches)
+if   [ -z "$CI" ] \
+  && [ -n "$1" ] \
+  && [ "$3" = "1" ] \
+  && [ "$1" != "$3" ]; \
+then
+  NO_FORMAT="\033[0m"
+  C_YELLOW="\033[38;5;11m"
```

Comment:
This color is hard to read on a light background terminal.  Not sure how much we want to get into this but assuming we should try to abide by accessibility principles here too, not quite sure how to handle this given a people can customize their terminal experiences, maybe a dark background on the text block itself?


Maybe something along these lines would work in both light and dark user preferences?
![image](https://user-images.githubusercontent.com/13666/218567962-53df6731-01bc-495b-a040-00961800898c.png)

Reply:
/cc @nikkimk

Reply:
these are ansi colours, so i think it's up to the term theme